web/src/app/shared/configuration/fields/fields.component.ts

```diff
@@ -57,14 +57,14 @@ export class ConfigFieldsComponent {
   constructor(private service: FieldService) {}
 
   get attr() {
-    return this.dataOptions.filter((a) => a.type === 'group' && (a as PanelOptions).activatable).reduce((p, c: PanelOptions) => ({ ...p, [c.name]: { active: c.active } }), {});
+    return this.dataOptions.filter((a) => a.type === 'group' && (a as IPanelOptions).activatable).reduce((p, c: IPanelOptions) => ({ ...p, [c.name]: { active: c.active } }), {});
   }
 
-  isPanel(item: FieldOptions | PanelOptions) {
+  isPanel(item: TFormOptions) {
     return 'options' in item && !item.hidden;
   }
 
-  trackBy(index: number, item: PanelOptions): string {
+  trackBy(index: number, item: IPanelOptions): string {
```

Comment:
It seems that the index is not needed

Reply:
https://angular.io/guide/structural-directives#inside-ngfor